### ISSUE TITLE
There's no local emulators suite on firebase-cli

### ISSUE THREAD

**Issue by francisrod01** [firebase/firebase-tools#2355]
### [REQUIRED] Environment info

**firebase-tools:** 8.2.0

**Platform:** Linux debian 4.9.0-12-amd64

### [REQUIRED] Steps to reproduce

```sh
~$ firebase init emulators -y
```

Then write down the `UI` lines in the `firebase.json`, connected on 4000 port.
A related issue here: https://github.com/firebase/firebase-tools/issues/2332

```js
"ui": {
      "enabled": true,      // Default is `true`
      "host": "localhost",
      "port": 4000          // If unspecified, see CLI log for selected port
    },
```

Firebase official doc: https://firebase.google.com/docs/emulator-suite/install_and_configure#configure_emulator_suite

After that, start the emulators.

```sh
~$ firebase emulators:start
```

### [REQUIRED] Expected behaviour

Is expected to show the Emulators Suite in UI mode.

![image](https://user-images.githubusercontent.com/4691768/84419197-a494c200-abee-11ea-984d-b8c517796361.png)

### [REQUIRED] Actual behavior

There's no Emulators Suite UI on the console, but only hub:

![image](https://user-images.githubusercontent.com/4691768/84419325-d73eba80-abee-11ea-8161-a375b9abe92d.png)

![image](https://user-images.githubusercontent.com/4691768/84419454-02290e80-abef-11ea-9de9-6a26202eaade.png)

### Firebase-cli logs using --debug flag

```
$ ./node_modules/.bin/firebase emulators:start --debug
[2020-06-11T17:23:00.445Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[2020-06-11T17:23:00.446Z] > authorizing via signed-in user
i  emulators: Starting emulators: functions, firestore, hosting
[2020-06-11T17:23:00.497Z] [hub] writing locator at /tmp/hub-medina-app-staging-67303.json
✔  hub: emulator hub started at http://localhost:4400
✔  functions: Using node@10 from host.
✔  functions: functions emulator started at http://localhost:5001
[2020-06-11T17:23:00.917Z] Ignoring unsupported arg: projectId
[2020-06-11T17:23:00.918Z] Ignoring unsupported arg: auto_download
[2020-06-11T17:23:00.918Z] Starting firestore emulator with command {"binary":"java","args":["-Duser.language=en","-jar","/home/paneladm/.cache/firebase/emulators/cloud-firestore-emulator-v1.11.3.jar","--host","localhost","--port",8080,"--rules","/home/paneladm/projects/paneladm-dev/medina-app/firestore.rules","--functions_emulator","localhost:5001"],"optionalArgs":["port","webchannel_port","host","rules","functions_emulator","seed_from_export"],"joinArgs":false}
i  firestore: firestore emulator logging to firestore-debug.log

i  firestore: firestore emulator logging to firestore-debug.log
[2020-06-11T17:23:02.124Z] API endpoint: http://localhost
[2020-06-11T17:23:02.125Z] :8080
If you are using a library that supports the FIRESTORE_EMULATOR_HOST environment variable, run:

   export FIRESTORE_EMULATOR_HOST=localhost:8080

Dev App Server is now running.


✔  firestore: firestore emulator started at http://localhost:8080
i  firestore: For testing set FIRESTORE_EMULATOR_HOST=localhost:8080
[2020-06-11T17:23:02.194Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/medina-app-staging-67303/webApps/-/config  
 
[2020-06-11T17:23:03.196Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Thu, 11 Jun 2020 17:23:02 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
i  hosting[default]: Serving hosting files from: dist/public
✔  hosting[default]: Local server: http://localhost:5000
i  hosting[staging]: Serving hosting files from: dist/public
✔  hosting[staging]: Local server: http://localhost:5005
✔  hosting: hosting emulator started at http://localhost:5000
i  functions: Watching "/home/paneladm/projects/paneladm-dev/medina-app/dist/functions" for Cloud Functions...
[2020-06-11T17:23:03.278Z] [worker-pool] addWorker(~diagnostic~) 
[2020-06-11T17:23:03.279Z] [worker-pool] Adding worker with key ~diagnostic~, total=1 
[2020-06-11T17:23:03.279Z] [worker-pool] submitWork(triggerId=) 
[2020-06-11T17:23:03.280Z] [worker-~diagnostic~-097da58f-0d5d-4c2e-bbce-a4099abeb9db]: Assigning socketPath: /tmp/fire_emu_8048.sock 
[2020-06-11T17:23:03.280Z] [worker-~diagnostic~-097da58f-0d5d-4c2e-bbce-a4099abeb9db]: BUSY 
[2020-06-11T17:23:03.674Z] [runtime-status] [8048] Functions runtime initialized. {"cwd":"/home/paneladm/projects/paneladm-dev/medina-app/dist/functions","node_version":"10.20.1"} 
[2020-06-11T17:23:03.675Z] [runtime-status] [8048] Disabled runtime features: undefined 
[2020-06-11T17:23:03.678Z] [runtime-status] [8048] Resolved module firebase-admin {"declared":true,"installed":true,"version":"8.11.0","resolution":"/home/paneladm/projects/paneladm-dev/medina-app/dist/functions/node_modules/firebase-admin/lib/index.js"} 
[2020-06-11T17:23:03.679Z] [runtime-status] [8048] Resolved module firebase-functions {"declared":true,"installed":true,"version":"3.6.1","resolution":"/home/paneladm/projects/paneladm-dev/medina-app/dist/functions/node_modules/firebase-functions/lib/index.js"} 
[2020-06-11T17:23:03.683Z] [runtime-status] [8048] Outgoing network have been stubbed. [{"name":"http","status":"mocked"},{"name":"http","status":"mocked"},{"name":"https","status":"mocked"},{"name":"https","status":"mocked"},{"name":"net","status":"mocked"}] 
[2020-06-11T17:23:03.723Z] [runtime-status] [8048] Checked functions.config() {"config":{}} 
[2020-06-11T17:23:03.724Z] [runtime-status] [8048] Resolved module firebase-functions {"declared":true,"installed":true,"version":"3.6.1","resolution":"/home/paneladm/projects/paneladm-dev/medina-app/dist/functions/node_modules/firebase-functions/lib/index.js"} 
[2020-06-11T17:23:03.725Z] [runtime-status] [8048] Resolved module firebase-admin {"declared":true,"installed":true,"version":"8.11.0","resolution":"/home/paneladm/projects/paneladm-dev/medina-app/dist/functions/node_modules/firebase-admin/lib/index.js"} 
[2020-06-11T17:23:03.726Z] [runtime-status] [8048] Resolved module firebase-functions {"declared":true,"installed":true,"version":"3.6.1","resolution":"/home/paneladm/projects/paneladm-dev/medina-app/dist/functions/node_modules/firebase-functions/lib/index.js"} 
[2020-06-11T17:23:03.726Z] [runtime-status] [8048] firebase-admin has been stubbed. {"adminResolution":{"declared":true,"installed":true,"version":"8.11.0","resolution":"/home/paneladm/projects/paneladm-dev/medina-app/dist/functions/node_modules/firebase-admin/lib/index.js"}} 
[2020-06-11T17:23:03.820Z] [runtime-status] [8048] initializeApp(DEFAULT) {"databaseURL":"https://medina-app-staging-67303.firebaseio.com","storageBucket":"medina-app-staging-67303.appspot.com","projectId":"medina-app-staging-67303"} 
[2020-06-11T17:23:03.830Z] addFirestoreTrigger "{\"eventTrigger\":{\"resource\":\"projects/medina-app-staging-67303/databases/(default)/documents/messages/{documentId}\",\"eventType\":\"providers/cloud.firestore/eventTypes/document.create\",\"service\":\"firestore.googleapis.com\"}}"
[2020-06-11T17:23:03.831Z] >>> HTTP REQUEST PUT http://localhost:8080/emulator/v1/projects/medina-app-staging-67303/triggers/makeUppercase  
 {"eventTrigger":{"resource":"projects/medina-app-staging-67303/databases/(default)/documents/messages/{documentId}","eventType":"providers/cloud.firestore/eventTypes/document.create","service":"firestore.googleapis.com"}}
[2020-06-11T17:23:03.832Z] [worker-~diagnostic~-097da58f-0d5d-4c2e-bbce-a4099abeb9db]: IDLE 
[2020-06-11T17:23:03.882Z] Jun 11, 2020 2:23:03 PM io.gapi.emulators.netty.HttpVersionRoutingHandler channelRead
INFO: Detected non-HTTP/2 connection.

[2020-06-11T17:23:04.077Z] Jun 11, 2020 2:23:04 PM io.gapi.emulators.netty.HttpVersionRoutingHandler channelRead
INFO: Detected HTTP/2 connection.

[2020-06-11T17:23:04.918Z] <<< HTTP RESPONSE 200 {"content-type":"application/json","content-length":"4"}
[2020-06-11T17:23:04.919Z] addFirestoreTrigger "{\"eventTrigger\":{\"resource\":\"projects/medina-app-staging-67303/databases/(default)/documents/customers/{customerId}\",\"eventType\":\"providers/cloud.firestore/eventTypes/document.create\",\"service\":\"firestore.googleapis.com\"}}"
[2020-06-11T17:23:04.919Z] >>> HTTP REQUEST PUT http://localhost:8080/emulator/v1/projects/medina-app-staging-67303/triggers/onCustomerCreated  
 {"eventTrigger":{"resource":"projects/medina-app-staging-67303/databases/(default)/documents/customers/{customerId}","eventType":"providers/cloud.firestore/eventTypes/document.create","service":"firestore.googleapis.com"}}
[2020-06-11T17:23:04.931Z] Jun 11, 2020 2:23:04 PM io.gapi.emulators.netty.HttpVersionRoutingHandler channelRead
INFO: Detected non-HTTP/2 connection.

[2020-06-11T17:23:04.958Z] <<< HTTP RESPONSE 200 {"content-type":"application/json","content-length":"4"}
✔  functions[nextApp]: http function initialized (http://localhost:5001/medina-app-staging-67303/us-central1/nextApp).
✔  functions[addMessage]: http function initialized (http://localhost:5001/medina-app-staging-67303/us-central1/addMessage).
✔  functions[makeUppercase]: firestore function initialized.
✔  functions[onCustomerCreated]: firestore function initialized.
✔  functions[getSearchKey]: http function initialized (http://localhost:5001/medina-app-staging-67303/us-central1/getSearchKey).
✔  emulators: All emulators started, it is now safe to connect.
```

**Comment by francisrod01** [comment]
Googling a while I found a solution not described on the docs. https://github.com/firebase/firebase-tools/issues/2272#issuecomment-636464586

It works if we update `firebase-tools` above `8.4.*`.

![image](https://user-images.githubusercontent.com/4691768/84421404-e7a46480-abf1-11ea-8e5f-b53b025c1a25.png)

### Update the docs

Could someone update the emulator's section in the docs?

**Comment by samtstern** [comment]
@francisrod01 yep the emulator suite was released in 8.4!
